src/averageAges.js

```diff
@@ -15,6 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  const arrOfSelectedMen = people.filter(person =>
+    century ? person.sex === 'm'
+      && Math.ceil(person.died / 100) === century : person.sex === 'm');
+
+  const sumOfAge = arrOfSelectedMen.reduce((sum, person) =>
+    sum + (person.died - person.born), 0);
```

Comment:
create a function for this calculations, you repeat the same logic several times